components/governance/org.wso2.carbon.governance.registry.extensions/src/org/wso2/carbon/governance/registry/extensions/indexers/RXTIndexer.java

```diff
@@ -70,6 +75,7 @@ public IndexDocument getIndexedDocument(AsyncIndexer.File2Index fileData) throws
             } else {
                 fields.put("overview_name", Arrays.asList(RegistryUtils.getResourceName(fileData.path).toLowerCase()));
             }
+            updateTenantsUnboundedFieldMap(configuration.getContentDefinition().getParent().toString());

```

Comment:
Hi Chandana,

Thanks for the suggestion. Change is done this commit [1]

[1] - https://github.com/wso2/carbon-governance/pull/194/commits/e4de82f55d9a5bbfc94a9cf8246b9d9099323937